internal/agent/device/status/exporters_non_linux.go

```diff
@@ -0,0 +1,45 @@
+//go:build darwin || !linux
```

Comment:
Need to rename the file to `exporters_other.go` for the conditional build to work on Darwin.

Reply:
wild :) no idea why